python/python/glide/async_commands/server_modules/json.py

```diff
@@ -214,6 +226,107 @@ async def forget(
     )
 
 
+async def strappend(
+    client: TGlideClient,
+    key: TEncodable,
+    value: TEncodable,
+    path: Optional[TEncodable] = None,
+) -> TJsonResponse[int]:
+    """
+    Appends the specified `value` to the string stored at the specified `path` within the JSON document stored at `key`.
+
+    See https://redis.io/commands/json.strapped/ for more details.
+
+    Args:
+        client (TRedisClient): The Redis client to execute the command.
+        key (TEncodable): The key of the JSON document.
+        value (TEncodable): The value to append to the string. Must be wrapped with single quotes. For example, to append "foo", pass '"foo"'.
+        path (Optional[TEncodable]): The JSONPath to specify. Default is root `$`.
```

Comment:
```suggestion
        path (Optional[TEncodable]): Represents the path within the JSON document where the string value will be appended. Default is root `$`.
```
To be consistent.  Although, this description is much too verbose.  I'd prefer that we go with something concise that mirrors the command description, like: 
```
Path within the JSON document.  If not specified, the root (`$`) is used. 
```

Reply:
fixed all of them, added Default to None. since the documentation say its `$` but its actually `.` so I better not specify whats exactly it is